users/templates/mail/welcome_and_next_steps.html

```diff
@@ -1,45 +1,75 @@
-    </style>
-</head>
+    <style type="text/css"></style>
+  </head>
 
-<body style="margin:0; padding:0; background-color:#F2F2F2;">
+  <body style="margin:0; padding:0; background-color:#F2F2F2;">
     <center>
-        <table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#F2F2F2">
-            <tr>
-                <td align="center" valign="top">
-                    <h1>Welcome {{user.first_name}} to Tampere Hacklab ry</h1>
-                    <p>
-                        Great to have you onboard! The Best way to get started is to join our community meeting first Tuesday of the month or weekly open house every Tuesday. It is also highly recommendable to join our communication channels (Matrix, Telegram or IRC and email list). Be happy to ask if you like to know more!
-                    </p>
-
-                    <h2>Payment info</h2>
-                    <p>Your personal bank reference number is <b>{{user.reference_number}}</b>. Remember always to use
-                        your reference number</p>
-                    <p>We will be waiting your first payment before we add your account access in to the systems. Before this you can join our community meetings, open house and communication channels and ask if someone could help you. All Hacklab services work by voluntary basis so things might not work always. Please be patient. For the challenges help would be appreciated, and we would be more than happy for your contribution.</p>
-
-                    <h3>Bank account information is</h3>
-
-                    <p><b>Account holder</b>: {{settings.ACCOUNT_NAME}}<br />
-                        ACCOUNT_IBAN : {{settings.IBAN}}<br />
-                        ACCOUNT_BIC : {{settings.BIC}}<br />
-                        ref : {{user.reference_number}}</p>
-						
-			<p>Best regards Tampere Hacklab hackers!</p>
-
-                </td>
-            </tr>
-        </table>
-    </center>
-</body>
+      <table
+        width="100%"
+        border="0"
+        cellpadding="0"
+        cellspacing="0"
+        bgcolor="#F2F2F2"
+      >
+        <tr>
+          <td align="center" valign="top">
+            <h1>Welcome {{user.first_name}} to {{ settings.SITENAME }}</h1>
+
+            <p>
+              Great to have you onboard! Your membership application has been
+              approved and now is a great time to to join our community by
+              coming to the next community meeting on the first Tuesday of the
+              month or poping by weekly open house every Tuesday. It is also
+              highly recommendable to join our communication channels (Matrix,
+              Telegram or IRC and email list). Be happy to ask if you like to
+              know more!
+            </p>
+
+            <h2>Payment info</h2>
+            <p>
+              Your personal bank reference number is
+              <b>{{user.reference_number}}</b>. Remember always to use your
+              reference number.
```

Comment:
Nope, this is not true. Personal number is used _only_ for default service (= access rights) payments, no others. Actually personal reference number should be hidden if user is not subscribed to access right.

Reply:
https://github.com/TampereHacklab/mulysa/pull/103/commits/51af7272cf788c964be2e2e362a4292c81d9b97c

now the reference number only shows for the default service